packages/next-frontend/src/app/[locale]/layout.tsx

```diff
@@ -29,7 +29,8 @@ export default async function RootLayout({
     const messages = await getMessages();
 
     return (
-        <html lang={locale}>
+        // TODO: use theme color
+        <html lang={locale} className="bg-[#F7F7F8]">
```

Comment:
No color in the Tailwind theme matching this?

Reply:
No, I've left it this way because I wasn't sure this is the right color to use as the background; anyway I can add this to the tailwind custom theme.